server/src/main/java/org/elasticsearch/action/support/replication/TransportBroadcastReplicationAction.java

```diff
@@ -126,7 +129,8 @@ protected void shardExecute(Task task, Request request, ShardId shardId, ActionL
         for (String index : concreteIndices) {
             IndexMetaData indexMetaData = clusterState.metaData().getIndices().get(index);
             if (indexMetaData != null) {
-                for (IntObjectCursor<IndexShardRoutingTable> shardRouting : clusterState.getRoutingTable().indicesRouting().get(index).getShards()) {
+                for (IntObjectCursor<IndexShardRoutingTable> shardRouting :
+                    clusterState.getRoutingTable().indicesRouting().get(index).getShards()) {
```

Comment:
I did this kind of thing as

```
for (IntObjectCursor<IndexShardRoutingTable> shardRouting
     : clusterState.getRoutingTable().indicesRouting().get(index).getShards()) {
```

Since I thought it made it more visually obvious that the second line was part of the `for` and not the code block below, but I could see either way.

Reply:
+1. I *really* like indenting an extra level for things like this so the for loop's "for" parts don't line up with the block's body.